framework/wazuh/exception.py

```diff
@@ -461,17 +461,17 @@ class WazuhException(Exception):
         4000: {'message': "Permission denied",
                'remediation': "Please, make sure you have permissions to execute the current request. "
                               "For more information on how to set up permissions, please visit XXXX"},
-        4001: {'message': 'The body of the request is empty, you must specify that you want to modify',
+        4001: {'message': 'The body of the request is empty, you must specify what you want to modify',
                'remediation': "The fields available for update are: name(str), rule(str), policies(list(dict))"},
         4002: {'message': 'The specified role does not exist',
                'remediation': 'Please, create the specified role with the endpoint POST /security/roles'},
         4003: {'message': 'The specified rule is invalid',
-               'remediation': "The rule must have a json format"},
+               'remediation': "The rule must be in json format."},
         4004: {'message': 'The specified name is invalid'},
         4005: {'message': 'The specified name or rule already exist'},
         4006: {'message': 'The specified policy is invalid',
-               'remediation': 'The policy must have a json format and it\'s keys must be "access", "resources", '
-                              'and "effect". The actions and resources must be splitted by ":". Example: agent:id:001'},
+               'remediation': 'The policy must be in json format and its keys must be "access", "resources" and'
+                              ' "effect". The actions and resources must be splitted by ":". Example: agent:id:001'},
```

Comment:
```suggestion
               'remediation': 'The policy must be in JSON format and its keys must be "access", "resources" and'
                              ' "effect". The actions and resources must be split by ":". Example: agent:id:001'},
```
We should probably take a look at every `json` to make it uppercase.